science/projects/zahrasadat_sajjadi/01-main-content/zarebin_p1.ipynb

```diff
@@ -0,0 +1,2717 @@
+{
```

Comment:
<p>using a context manager instead of closing manaully? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Zarebin/nextgen-06/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>